dataprofiler/profilers/text_column_profile.py

```diff
@@ -56,8 +56,8 @@ def __add__(self, other: TextColumn) -> TextColumn:
                 "'TextColumn' and '{}'".format(other.__class__.__name__)
             )
         merged_profile = TextColumn(None)
-        NumericStatsMixin._add_helper(merged_profile, self, other)
         BaseColumnPrimitiveTypeProfiler._add_helper(merged_profile, self, other)
+        NumericStatsMixin._add_helper(merged_profile, self, other)
```

Comment:
I need to think about how this order could have affected other things, but tests do pass.

Reply:
So, im not sure if there was a deliberate reason to have this order not match the other profiles that use these two functions, but  the other two (float and int) do it in the updated order you see in the change. Previously order didn't matter so that might have been why it differed and no one said anything in the PR.

Reply:
Looking farther into this, this specific line was implemented on release 3 years ago, so I wouldn't really know the reasoning behind the order differing decision as thats before my time...